src/python/grpcio_tests/tests/_sanity/_sanity_test.py

```diff
@@ -38,6 +38,8 @@ def testTestsJsonUpToDate(self):
         tests_json_string = pkg_resources.resource_string('tests', 'tests.json')
         tests_json = json.loads(tests_json_string.decode()
```

Comment:
It's weird that there's an "if six.PY3" on the line above an "if six.PY2", and of course we shouldn't be reassigning the tests_json local field if we can avoid it. Should this logic just be expressed in a simple if/else?

Reply:
I've reverted this file.  The changes were not needed.